src/org/thoughtcrime/securesms/conversation/ConversationFragment.java

```diff
@@ -414,7 +417,8 @@ private void setCorrectMenuVisibility(Menu menu) {
       menu.findItem(R.id.menu_context_reply).setVisible(isPublicChat && selectedMessageCount == 1);
       String userHexEncodedPublicKey = TextSecurePreferences.getLocalNumber(getContext());
       boolean userCanModerate = isPublicChat && LokiPublicChatAPI.Companion.isUserModerator(userHexEncodedPublicKey, publicChat.getChannel(), publicChat.getServer());
-      boolean isDeleteOptionVisible = isPublicChat && selectedMessageCount == 1 && (isSentByUser || userCanModerate);
+//      boolean isDeleteOptionVisible = isPublicChat && selectedMessageCount == 1 && (isSentByUser || userCanModerate);
+      boolean isDeleteOptionVisible = isPublicChat && (isSentByUser || userCanModerate);
```

Comment:
`isSentByUser` now doesn't make sense anymore. We should either remove the flag or update it to make sense again.